sdk-api-src/content/d3d12/ns-d3d12-d3d12_query_data_so_statistics.md

```diff
@@ -56,16 +56,16 @@ Describes query data for stream output.
 
 ### -field NumPrimitivesWritten
 
-Specifies the number of primitives written.
+Specifies the number of primitives actually written to the stream output resource.
 
 ### -field PrimitivesStorageNeeded
 
-Specifies the total amount of storage needed by the primitives.
+Specifies the total amount of storage needed by all the primitives that would have been written to the stream output resource if it was big enough to do so.
```

Comment:
This change in general looks good. 

We could go an extra mile and re-phrase this to say what units the storage is in (bytes? bits? number of primitives? answer: Number of primitives). This is something the [11 doc](https://docs.microsoft.com/en-us/windows/win32/api/d3d11/ns-d3d11-d3d11_query_data_so_statistics) is crisp and gets it right, while the 12 doc is vague.

Reply:
Thanks, @clandrew! I'll go ahead and make that change in a separate PR.

Reply:
Thanks guys, this makes it much more clear what's going on!